etc/conda_build_matrix.py

```diff
@@ -74,11 +80,21 @@ def main(upload, upload_only, user):
                     err.decode('utf-8'),
                 ))
         elif upload:
-            files.append(subprocess.Popen(
+            p = subprocess.Popen(
                 **mkargs(py_version, npy_version, output=True)
```

Comment:
How does this differ from the functionality of `proc`?  Are we `conda build`ing twice?